### ISSUE TITLE
How does the code combine 9 images in different brightness into 1 image before fitting in the model?

### ISSUE THREAD

**Issue by rjchien728** [yhenon/pytorch-retinanet#53]
In  visualize.py line 68, I save data['img'] before it fit to the model.
and find out the origin image was not only processed with Normalization and Resize,
but also done some brightness adjust and put 9 of them in 3 by 3 grid together.

However I can not figure out how and where does it be done.
I need to know how to do it cause I want to implement a real time webcam prediction demo.

Also, does it mean during the training the model is actually watching only one third resolution even after resized?

Thanks for the answer.

**Comment by rjchien728** [comment]
I found out that this operation is done by collater(data) in dataloader.py
